doc/manual/metaprogramming.rst

```diff
@@ -545,7 +545,7 @@ This macro can be used like this:
     julia> @assert 1==1.0
 
     julia> @assert 1==0
-    ERROR: AssertionError: 1 == 0
+    ERROR: Assertion failed: 1 == 0
```

Comment:
I have tried to modify the example as you suggested and make the rest of the document be consistent with the modification. Hope it works...